explore_data/nbconverted/cluster_by_metadata.py

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Clustering by metadata
+# 
+# This notebook examines how the data clusters by metadata. We would expect that the data cluster by strain type (PAO1 and PA14).
+# 
+# Other metadata?
```

Comment:
It's unclear what this line means

Reply:
This was me thinking "outloud" of other metadata fields to consider